api/v1alpha1/ycsbbench_types.go

```diff
@@ -0,0 +1,53 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// YcsbBenchSpec defines the desired state of YcsbBench
+type YcsbBenchSpec struct {
+	// Image defines the docker image used for the benchmark
+	Image ImageSpec `json:"image"`
+
+	Database   string            `json:"database"`
+	Workload   string            `json:"workload"`
+	Options    YcsbBenchOptions  `json:"options,omitempty"`
```

Comment:
I presume this is optional field based on the omitempty. If that's the case please add // +optional directive.

Reply:
Done!